admin/people_request_builder.go

```diff
@@ -105,20 +83,6 @@ func (m *PeopleRequestBuilder) ToGetRequestInformation(ctx context.Context, requ
     requestInfo.Headers.TryAdd("Accept", "application/json")
     return requestInfo, nil
 }
-// ToPatchRequestInformation update the navigation property people in admin
```

Comment:
updating is not supported https://learn.microsoft.com/en-us/graph/api/peopleadminsettings-get?view=graph-rest-1.0&tabs=http